src/exampleTypes/ResultTable.js

```diff
@@ -23,7 +23,7 @@ let Header = withStateLens({ popover: false })(
     <th>
       <a onClick={F.flip(popover)}>
         {label}{' '}
-        {field === node.sortField && (node.sortDir === 'asc' ? '^' : 'v')}
+        {field === node.sortField && (node.sortDir === 'asc' ? '↑' : '↓')}
```

Comment:
Neat :) The long term plan is to figure out how to expose this to be customized by consumers but this is an improvement